test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -853,6 +877,9 @@ redis_query(Node, Query) ->
     RedisWorker = rpc(Node, wpool_pool, best_worker, [mod_global_distrib_mapping_redis]),
     rpc(Node, eredis, q, [RedisWorker, Query]).
 
+advertised_endpoints() ->
+    [{ {231, 110, 1, 4}, 2222}].
```

Comment:
Comment: This address is a fake one, we never try to connect there.
Used in test_advertised_endpoints_override_endpoints test case.